virt/virtual_machines/advanced_vm_management/virt-enabling-descheduler-evictions.adoc

```diff
@@ -7,12 +7,17 @@ include::_attributes/common-attributes.adoc[]
 toc::[]
 
 :FeatureName: Descheduler eviction for virtual machines
-include::snippets/technology-preview.adoc[]
 
-The xref:../../../nodes/scheduling/nodes-descheduler.adoc#nodes-descheduler[descheduler] can be used to evict a running pod to allow the pod to be rescheduled onto a more suitable node. You must install the descheduler by using the {product-title} web console or OpenShift CLI (`oc`) before you can enable it on your virtual machine (VM).
+You can evict a running pod with the descheduler in order to schedule the pod on a more suitable node. If the pod is a virtual machine, the pod eviction causes the virtual machine to be live migrated to another node.
```

Comment:
> You can evict a running pod with the descheduler in order to schedule the pod on a more suitable node.

Where the scheduler does the scheduling, not the user or descheduler, perhaps:

"You can evict a running pod with the descheduler to allow the pod to be scheduled on a more suitable node."

wdyt?

Reply:
I see what you're saying. To avoid passive voice, how about this: 

"You can evict a running pod with the descheduler, which allows the scheduler to schedule the pod on a more suitable node."